plugin/src/main/java/net/dzikoysk/funnyguilds/user/UserRank.java

```diff
@@ -0,0 +1,125 @@
+package net.dzikoysk.funnyguilds.user;
+
+import net.dzikoysk.funnyguilds.rank.Rank;
+
+public class UserRank extends Rank {
+
+    private final User user;
+
+    private int points;
+    private int kills;
+    private int deaths;
+    private int assists;
+    private int logouts;
+
+    public UserRank(User user, int rankStart) {
+        super(user);
+
+        this.user = user;
+        this.points = rankStart;
+    }
+
+    public User getUser() {
+        return user;
+    }
+
+    @Override
+    public int getPoints() {
+        return this.points;
+    }
+
+    public void setPoints(int points) {
+        this.points = Math.max(0, points);
+        this.entity.markChanged();
+    }
+
+    public void addPoints(int change) {
```

Comment:
Proponuję dodać sprawdzanie, czy przy add i remove jest używana dodania liczba i robić throw przy niepoprawnej wartości.
Zakładając, że system rankingu działa dobrze to nie powinno wyliczyć wartości ujemnej do dodawania lub odejmowania rankingu.
W przypadku kiedy zwycięzca dostanie np. -10pkt to lepiej już wywalić błąd niż ignorować to.